### DISCUSSION TITLE
RP6502_VGA Picoprobe firmware is out of date. Any plans to update?

### DISCUSSION THREAD

**Discussion by vruumllc** [picocomputer/community#43] [Q&A]
Hi, 
It looks like the RP6502_VGA src incorporates the old "picoprobe" descriptors rather than the current release that looks like CMIS_DAPv2, requiring the old hacked OpenOCD to do debugging of the rp6502 firmware. 

Any plans to update to the newest version?

Thanks

**Comment by vruumllc** [comment]
Hmm, to answer my own question --Looking at the new picoprobe code, I see it now uses FreeRTOS, which is a major change, and probably screws up your architecture.

I guess I'll just build the old hacked version of OpenOCD, and see if I can get both versions to coexist on my laptop.

**Comment by rumbledethumps** [comment]
https://github.com/picocomputer/rp6502/issues/29
